draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2067,6 +2067,94 @@ the Waiting state, performs a connectivity check on that pair and puts the
 candidate pair state to In-Progress.
 </t>
 </list></t>
+<t>
+The flow chart in <xref target="fig-cc-schedule"/> illustrates the procedures
+above.
+</t>
+
+<figure title="Scheduling Connectivity Checks" anchor="fig-cc-schedule" align="center"><artwork>
+<![CDATA[
+
+      BEGIN (Ta fires)
+             |
+             v
++==========================+
+|Select a CHECK LIST (from |
+|the CHECK LIST SET) to    |<--------------------------------------+
+|process.                  |                                       |
++==========================+                                       |
+             |                                                     |
+             v                                                     |
++--------------------------+        +==========================+   |
+|Candidate Pair(s) in      |        |Remove top-most Candidate |   |
+|TRIGGERED CHECK QUEUE     |   YES  |Pair from the TRIGGERED   |   |
+|associated with selected  |------->|CHECK QUEUE, set the pair |   |
+|CHECK LIST?               |        |state to In-Progress and  |   |
+|                          |        |perform connectivity check|   |
+|                          |        |on the pair. Restart Ta.  |   |
++--------------------------+        +==========================+   |
+             |                                                     |
+             | NO                                                  |
+             v                                                     |
++--------------------------+                                       |
+|Candidate Pair(s) in      |   YES                                 |
```

Comment:
For "YES", this looks like it goes back to "select a check list", creating an endless loop?